kvm.yaml

```diff
@@ -69,6 +106,23 @@
       version: 0.2.0
     - name: kvm-operator
       version: 3.10.0
+- version: 9.3.0
+  active: true
+  date: 2020-03-02T12:00:00Z
+  authorities:
+    - name: app-operator
+      version: 1.0.0
+    - name: cert-operator
+      version: 0.1.0
+    - name: chart-operator
+      version: 0.7.0
+    - name: cluster-operator
+      provider: kvm
+      version: 0.21.0
+    - name: flannel-operator
+      version: 0.2.0
+    - name: kvm-operator
+      version: 3.9.0
```

Comment:
This is tough. Which should be the value here?

Reply:
Keep the `-dev` version you had previously for now - I want to at least create a cluster and check some apps.

Reply:
Sorry the previous comment was meant to be on line 37.

Here we will need to do some more magic. I think this will require some changes in `thiccc` branch of kvm-operator. Let's talk tomorrow.